crates/workspace/src/pane_group.rs

```diff
@@ -720,27 +723,49 @@ impl PaneAxis {
         render_cx: &dyn PaneLeaderDecorator,
         window: &mut Window,
         cx: &mut App,
-    ) -> gpui::AnyElement {
+    ) -> (gpui::AnyElement, bool) {
```

Comment:
Let's introduce a regular `struct` with named fields instead of this newtype, so it's more clear what that `bool` means.